students/piotrowski_stanislaw/lesson_03_functions/calculator.py

```diff
@@ -0,0 +1,88 @@
+# Calculator.py
+# piotrsta
+
+
+def help_calc():
+    print("a - add")
+    print("s - subtract")
+    print("m - multiply")
+    print("d - divide")
+    print("p - power")
+    print("h,? - help")
+    print("q - QUIT")
+
+
+def add(x, y):
+    print("Result: " + str(x + y))
+
+
+def subtract(x, y):
+    print("Result: " + str(x - y))
+
+
+def multiply(x, y):
+    print("Result: " + str(x * y))
+
+
+def divide(x, y):
+    print("Result: " + str(x / y))
+
+
+def power(x, y):
+    print("Result: " + str(x ** y))
+
+
+def input_operand1():
+    operand1 = int(input("Input 1st operand: "))
+    return operand1
+
+
+def input_operand2():
+    operand2 = int(input("Input 2nd operand: "))
+    return operand2
+
+
+print("Welcome to NEW calculator:")
```

Comment:
całą tą część kodu, która już wywołuje kalkulator można by zamknąć w warunku:
`if name == 'main':`
Dzięki czemu podczas importu funkcji z tego modułu do innego modułu/skryptu ta część modułu nie zostanie wykonana.
On się wykona tylko wtedy, gdy ten moduł/skrypt zostanie uruchomiony jak skrypt (na najwyższym poziomie).
Jeżeli uruchomisz ten skrypt zmienna name przyjmie wartość 'main', jeżeli zaimportujesz jako moduł to name ma wartość: 'nazwa_pliku'.
Pamiętaj o tym tworząc nowy moduł i możesz to tez dodać do pozostałych zadań.
W skryptach z lekcji 0 uzyty był ten mechanizm, możesz zobaczyć jak go użyto.